File: app/src/main/java/com/arjanvlek/oxygenupdater/settings/BottomSheetItem.java
package com.arjanvlek.oxygenupdater.settings;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

/**
 * @author Adhiraj Singh Chauhan (github.com/adhirajsinghchauhan)
 */
@SuppressWarnings("WeakerAccess")
@Getter
@Setter
@Builder
public class BottomSheetItem {

	private String title;

	private String subtitle;

	private String value;

	private Object secondaryValue;
}

File: app/src/main/java/com/arjanvlek/oxygenupdater/settings/BottomSheetItem.kt
package com.arjanvlek.oxygenupdater.settings

import lombok.Builder
import lombok.Getter
import lombok.Setter

/**
 * @author Adhiraj Singh Chauhan (github.com/adhirajsinghchauhan)
 */
@Getter
@Setter
@Builder
class BottomSheetItem {

    var title: String? = null
        set(title) {
            field = this.title
        }

    var subtitle: String? = null
        set(subtitle) {
            field = this.subtitle
        }

    var value: String? = null
        set(value) {
            field = this.value
        }

    var secondaryValue: Any? = null
        set(secondaryValue) {
            field = this.secondaryValue
        }
}
